ecosystem/typescript/sdk/examples/typescript/transfer_coin.ts

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) Aptos
+// SPDX-License-Identifier: Apache-2.0
+
+/* eslint-disable no-console */
+
+import dotenv from "dotenv";
+dotenv.config();
+
+import { AptosClient, AptosAccount, CoinClient, FaucetClient } from "aptos";
+import { aptosCoinStore, NODE_URL, FAUCET_URL } from "./common";
+
+(async () => {
+  // Create API and faucet clients.
+  // :!:>section_1
+  const client = new AptosClient(NODE_URL);
+  const faucetClient = new FaucetClient(NODE_URL, FAUCET_URL); // <:!:section_1
+
+  // Create client for working with the coin module.
+  // :!:>section_1a
+  const coinClient = new CoinClient(client); // <:!:section_1a
+
+  // Create accounts.
+  // :!:>section_2
+  const alice = new AptosAccount();
+  const bob = new AptosAccount(); // <:!:section_2
+
+  // Print out account addresses.
+  console.log("=== Addresses ===");
+  console.log(`Alice: ${alice.address()}`);
+  console.log(`Alice: ${bob.address()}`);
+  console.log("");
+
+  // Fund accounts.
+  // :!:>section_3
+  await faucetClient.fundAccount(alice.address(), 20_000);
+  await faucetClient.fundAccount(bob.address(), 0); // <:!:section_3
+
+  // Print out initial balances.
+  console.log("=== Initial Balances ===");
+  // :!:>section_4
+  console.log(`Alice: ${await coinClient.checkBalance(alice)}`);
```

Comment:
nit: This can be more efficient with const [aliceBal, bobBal] = await Promise.all([coinClient.checkBalance(alice), coinClient.checkBalance(bob)]);

Reply:
Given this is just an example I'll opt to keep it clearer, but otherwise yeah that'd be better.